source/NVDAObjects/window/excel.py

```diff
@@ -944,7 +944,8 @@ class ExcelCellTextInfo(NVDAObjectTextInfo):
 
 	def _getFormatFieldAndOffsets(self,offset,formatConfig,calculateOffsets=True):
 		formatField=textInfos.FormatField()
-		if (self.obj.excelCellObject.Application.Version > "12.0"):
+		version=int(self.obj.excelCellObject.Application.Version.split('.')[0])
+		if version>12:

```

Comment:
Please add a comment, or better yet name 12 as `Office2010VersionMajor` or whatever it equates to.

Because this is comparing the major part, and we assume that 12.1 would not include the required feature, I think that the intent is clearer if we use a `version<13` comparison.